components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -0,0 +1,72 @@
+import { toUpper } from 'lodash'
+import React, { useState } from 'react'
+import { Tab, Col, Nav } from 'react-bootstrap'
+import styles from '../../../scss/adminLessonNav.module.scss'
+
+type AdminLessonNavProps = {
+  navItems: { value: string }[]
+  tabs: (() => JSX.Element)[]
+}
+
+const validateProps = ({ navItems, tabs }: AdminLessonNavProps) => {
+  if (navItems.length !== tabs.length)
+    throw new Error(
+      `navItems and tabs should have the same count. navItems: ${navItems.length} -- tabs: ${tabs.length}`
+    )
+
+  const mapObjArrayToValues = navItems.map(item => item.value)
+
+  if (new Set(mapObjArrayToValues).size !== mapObjArrayToValues.length)
```

Comment:
Instead of
```
const mapObjArrayToValues = navItems.map(item => item.value)
if (new Set(mapObjArrayToValues).size !== mapObjArrayToValues.length)
```
Can you do

```
if (_.uniq(navItems).length !== navItems.length) {
    ................
}
``` 

uniq function is from lodash library
https://lodash.com/docs/4.17.15#uniq

Reply:
Sure, that'd better.